styles.css

```diff
@@ -113,16 +114,287 @@
 
 .main__button {
   display: inline-block;
+  position: relative;
+  left: 8px;
   margin-top: 40px;
-  font: 18px Roboto, arial, helvetica, sans-serif;
+  font: 18px roboto, arial, helvetica, sans-serif;
   color: #fff;
   background: #0056ec;
-  padding: 18px 28px 18px 21px;
+  padding: 18px 30px 15px 23px;
   text-decoration: none;
   border-radius: 4px;
-  transition: opacity  .5s;
+  transition: opacity 1s;
 }
 
-.main__button:hover {
-  opacity: .5;
+.main__button-blic {
+    position: relative;
+    margin: 0;
+    width: 8px;
+    height: 70px;
+    background: #fff;
+    transform: rotate(-45deg);
+    opacity: .1;
+    animation: example 7s infinite ease-in-out alternate;
+}
+
+@keyframes example {
+    0%   {left:0px; top:-59px;}
+    50%  {left:210px; top:-59px;}
+    100% {left:0px; top:-59px;}
+}
+
+@media (max-width: 1500px) {
+  .wrapper {
+    width: 935px;
+  }
+
+  .header {
+    height: 34px;
+    margin-bottom: 120px;
+  }
+
+  .menu {
+    display: flex;
+  }
+
+  .menu__item {
+    margin-left: 24px;
+    font: 15px/51px roboto, helvetica, arial, sans-serif;
+  }
+
+  .menu__item_active::after {
+    display: block;
+    top: 35px;
+  }
+
+  .main {
+    margin-top: 0;
+    background: top right no-repeat url(img/frame1024.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 40px;
+    padding: 0;
+    font: 17px/20px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 41px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 1300px) {
+  .wrapper {
+    width: 93%;
+  }
+
+  .header {
+    height: 34px;
+    margin-bottom: 120px;
+  }
+
+  .menu {
+    display: flex;
+  }
+
+  .menu__item {
+    margin-left: 24px;
+    font: 15px/51px roboto, helvetica, arial, sans-serif;
+  }
+
+  .menu__item_active::after {
+    display: block;
+    top: 35px;
+  }
+
+  .main {
+    margin-top: 0;
+    background: top right no-repeat url(img/frame1024.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 40px;
+    padding: 0;
+    font: 17px/20px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 41px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 800px) {
+  .wrapper {
+    margin: auto;
+    width: 80%;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .logo__name {
+      opacity: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6320070)